### DISCUSSION TITLE
Thank you to the developers for adding the \QUALIFIER feature

### DISCUSSION THREAD

**Discussion by randolf** [doxygen/doxygen#10925] [Show and tell]
Recently I asked about adding custom qualifiers to method documentation (alongside the built-in qualifiers like inline, noexcept, static, etc.), and the \QUALIFIER feature was added pretty quickly.  I appreciate how well-supported Doxygen is, and how useful the generated documentation is.  Thank you to everyone who participates and contributes to Doxygen.

If you want to see how I'm using custom qualifiers, you can see the documentation for a project I've been working on, here:  https://www.randolf.ca/c++/docs/classrandolf_1_1rsocket.html

Thanks again, and I will continue to highly recommend Doxygen to everyone I know.

**Comment by paule32** [comment]
looks very good.
How did you render the Page Style - with XML Output of DoxyGen, and customized XSLT File ?

**Comment by cuppajoeman** [comment]
> Recently I asked about adding custom qualifiers to method documentation (alongside the built-in qualifiers like inline, noexcept, static, etc.), and the \QUALIFIER feature was added pretty quickly. I appreciate how well-supported Doxygen is, and how useful the generated documentation is. Thank you to everyone who participates and contributes to Doxygen.
> 
> If you want to see how I'm using custom qualifiers, you can see the documentation for a project I've been working on, here: https://www.randolf.ca/c++/docs/classrandolf_1_1rsocket.html
> 
> Thanks again, and I will continue to highly recommend Doxygen to everyone I know.

Looks good, I think I might have the same question as paule32, but can you share the files/configuration to get those styles for your doxygen docs?
